state/errors.go

```diff
@@ -29,15 +22,7 @@ var (
 	IsCharmAlreadyUploadedError    = stateerrors.IsCharmAlreadyUploadedError
 	IsProviderIDNotUniqueError     = stateerrors.IsProviderIDNotUniqueError
 	IsParentDeviceHasChildrenError = stateerrors.IsParentDeviceHasChildrenError
-	IsIncompatibleSeriesError      = stateerrors.IsIncompatibleSeriesError
 	IsNotAlive                     = stateerrors.IsNotAlive
-	IsHasAssignedUnitsError        = stateerrors.IsHasAssignedUnitsError
-	IsHasContainersError           = stateerrors.IsHasContainersError
-	IsHasAttachmentsError          = stateerrors.IsHasAttachmentsError
-	IsHasHostedModelsError         = stateerrors.IsHasHostedModelsError
-	IsHasPersistentStorageError    = stateerrors.IsHasPersistentStorageError
-	IsModelNotEmptyError           = stateerrors.IsModelNotEmptyError
-	IsStorageAttachedError         = stateerrors.IsStorageAttachedError
 	IsDeletedUserError             = stateerrors.IsDeletedUserError
```

Comment:
why do we still need some of these?

Reply:
Haven't looked into all of them yet. Was just about getting this to a logic point. Happy to give them some work in this PR if you think it should be done.